src/libos/src/net/socket/unix/stream/server.rs

```diff
@@ -0,0 +1,110 @@
+use super::*;
+use std::collections::btree_map::BTreeMap;
+
+#[derive(Debug)]
+pub struct Servers {
```

Comment:
I think this is better named `AddressSpace`. And `SERVER_SPACE` renamed as `ADDRESS_SPACE`.

Reply:
Second thought. `AddressRegistry` seems to be better.

Reply:
I still prefer address space. Name space is also ok to me. `Registry` is not a word in the posix context. I have never seen the word.